vllm/model_executor/models/gpt_bigcode.py

```diff
@@ -259,16 +271,37 @@ def __init__(
         self.quant_config = quant_config
         self.transformer = GPTBigCodeModel(config, cache_config, quant_config,
                                            lora_config)
+
+        self.unpadded_vocab_size = config.vocab_size
+        if lora_config:
+            self.unpadded_vocab_size += lora_config.lora_extra_vocab_size
+
         if self.config.tie_word_embeddings:
-            self.lm_head = self.transformer.wte
+            if quant_config is not None:
+                linear_method = quant_config.get_quant_method(self)
```

Comment:
Shouldn't the `get_quant_method` need `prefix` passed in as well?

Reply:
Yes